sungbin/section06/src/com/library/users/User.java

```diff
@@ -0,0 +1,34 @@
+package com.library.users;
+
+import com.library.books.Book;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class User {
+
+    private String name;
+
+    private List<Book> borrowedBooks = new ArrayList<>();
+
+    public User(String name) {
+        this.name = name;
+    }
+
+    public void borrow(Book book) {
+        borrowedBooks.add(book);
+        System.out.println(name + "님이 " + book.getTitle() + "의 책을 빌렸습니다.");
+    }
+
+    public void returnTo(Book book) {
+        borrowedBooks.remove(book);
+        System.out.println(name + "님이 " + book.getTitle() + "의 책을 반납했습니다.");
+    }
+    
+    public void showBorrowedBooks() {
+        System.out.println("=======" + name + "님이 빌린 책 목록 =======");
+        for (Book book : borrowedBooks) {
+            System.out.println(book.display());
+        }
+    }
```

Comment:
`display()` 메서드가 필요한 이유가 뭘까요?

Reply:
@crispindeity 솔직히 필요는 없습니다만, 요구조건에 Book 클래스를 만들고, 책 정보를 출력하는 메서드를 작성하세요. 라는 사항이 있어서 일부러 이렇게 하긴 했습니다!

Reply:
```suggestion
    public void showBorrowedBooks() {
        System.out.println("=======" + name + "님이 빌린 책 목록 =======");
        for (Book book : borrowedBooks) {
            System.out.println(book);
        }
    }
```
`toString()` 을 `Override` 했다면 `book.display()` 없이 `book` 만 넘겨줘도 같은 동작을 할 텐데 별도의 메서드를 사용하신 게 궁금했습니다. 사소하지만 메서드를 선언하여 사용하는것도 그에 따른 오버헤드가 발생하기 때문에 꼭 필요한 메서드가 아니라면 줄여보는것도 좋을것 같습니다.

Reply:
네네 퇴근하고 반영해보겠습니다!